### ISSUE TITLE
More examples

### ISSUE THREAD

**Issue by tomwhite** [cubed-dev/cubed#648]
One of the things that became apparent from AGU was that we need more examples!

Here are a few that I'd like to add:

* [x] Zarr. Read a regular Zarr array (using `from_zarr`), apply some array operations, then write the result out as Zarr (using `to_zarr`).
* [x] Xarray. Open a Zarr store as an Xarray Dataset, apply some array operations, then write the result out as Zarr.
* [x] Rechunk. Rechunk an Xarray Dataset. See #539 
* [x] Icechunk. Same as the above, but using Icechunk.
* [ ] VirtualiZarr + Rechunk. Combine some NetCDF files using VirtualiZarr, rechunk (using Cubed), then save. See #646
* [ ] Single cloud machine. How to run Cubed on a single (large) cloud instance where we read/write from cloud storage, but intermediate data is stored on SSD.
* [ ] https://github.com/cubed-dev/cubed/issues/519

**Comment by TomNicholas** [comment]
Yes to all of these! Also an example showing that xarray groupby with flox works.

**Comment by tomwhite** [comment]
> Yes to all of these! Also an example showing that xarray groupby with flox works.

We already have https://flox.readthedocs.io/en/latest/user-stories/climatology-hourly-cubed.html, but we could add something to the Cubed docs too - even if it's just a link to that page.
